AssistantV1.playground/Sources/Helpers.swift

```diff
@@ -16,6 +16,142 @@ public func setupAssistantV1() -> Assistant {
     return assistant
 }
 
+// Find or create a readWrite workspace
+public func getWorkspaceID() -> String {
+
+    let assistant = setupAssistantV1()
+
+    var workspaceID: String?
+    assistant.listWorkspaces() {
+        response, error in
+
+        guard let result = response?.result else {
+            assertionFailure(error?.localizedDescription ?? "unexpected error")
+            return
+        }
+
+        workspaceID = result.workspaces.first?.workspaceID
+        if workspaceID == nil {
+            assistant.createWorkspace(name: "my_environment", description: "My environment") {
+                response, error in
+
+                guard let workspace = response?.result else {
+                    assertionFailure(error?.localizedDescription ?? "unexpected error")
+                    return
+                }
+
+                workspaceID = workspace.workspaceID
+            }
+        }
+    }
+
+    while workspaceID == nil { sleep(1) }
+    return workspaceID!
+}
+
+public func createSampleIntent(workspaceID: String) {
+    let assistant = setupAssistantV1()
+    var created = false
+    assistant.getIntent(workspaceID: workspaceID, intent: "hello") {
+        _, error in
+
+        if let error = error {
+            if case let .http(statusCode, _, _) = error, statusCode == 404 {
+                // Not Found -- fall through to creation
+            } else {
+                print(error.localizedDescription)
+                return
+            }
+
+            assistant.createIntent(
+                workspaceID: workspaceID,
+                intent: "hello",
+                examples: [
+                    CreateExample(text: "Good afternoon"),
+                    CreateExample(text: "Hi there")
+                ])
+            {
+                _, error in
+
+                if let error = error {
+                    print(error.localizedDescription)
+                    return
+                }
+
+                created = true
+            }
+        }
+
+        created = true
+    }
+    while !created { sleep(1) }
+    return
+}
+
+public func deleteSampleIntent(workspaceID: String) {
+    let assistant = setupAssistantV1()
+    assistant.deleteIntent(workspaceID: workspaceID, intent: "hello") {
```

Comment:
To make this a little more robust, the "hello" string should be put into a constant to be used in both `createSampleIntent()` and `deleteSampleIntent()`.

Reply:
Will do.